stratum/hal/lib/tdi/dpdk/dpdk_sde_target.cc

```diff
@@ -203,6 +204,16 @@ ::util::Status TdiSdeWrapper::InitNotificationTableWithCallback(
          << "Notification Table not supported";
 }
 
+// PacketModMeter
+::util::Status TableData::SetPktModMeterConfig(
+    const TdiPktModMeterConfig& cfg) {
+  return MAKE_ERROR(ERR_UNIMPLEMENTED) << "Packet-mode meter not supported";
```

Comment:
typo. mode -> mod

Reply:
If `Mod` doesn't stand for _mode_, what does it stand for?

Reply:
Modify

Reply:
Fixed